packages/wrangler/e2e/dev-with-resources.test.ts

```diff
@@ -370,6 +370,10 @@ describe.sequential.each(RUNTIMES)("Bindings: $flags", ({ runtime, flags }) => {
 				name = "${workerName}"
 				main = "src/index.ts"
 				compatibility_date = "2025-01-01"
+				# Regression test for https://github.com/cloudflare/workers-sdk/issues/9006
+				kv_namespaces = [
+					${isLocal ? `{ binding = "KV", id = "LOCAL_ONLY" }` : ""}
+				]
```

Comment:
I ran the same test without the fix here: https://github.com/cloudflare/workers-sdk/actions/runs/15183927642/job/42699817229?pr=9338#step:5:4996